provisioning/atat_provisioning.yaml

```diff
@@ -1,21 +1,48 @@
 openapi: 3.0.2
 info:
   description: >-
-    This is a DRAFT version of the API meant for provisioning resources into a JWCC Provider Cloud Environment. It will be
-    implemented and maintained by all participating CSPs.
+    This API is meant for provisioning resources in a JWCC Provider Cloud Environment.
+    The cloud products and services used in JWCC are provided by commercial cloud service providers (CSP).
+    This API will be implemented and maintained by all participating CSPs.
+
+    ### Organization of provisioned resources
+    
+    Provisioned resources under the purview of a mission owner are organized in a three-level hierarchy.
+
+    * Portfolio
+      * Application
+        * Environment
+
+    *Portfolios* consist of one or more applications. *Applications* consist of one or more environments.
+    *Environments* directly contain provisioned resources.
+
+    Access can be granted to *operators* at each level.
+
+    ### Funding sources and cost reporting
+
+    Portfolios contain one or more funding sources. A funding source consists of a
+    *Task Order* (TO) and a *Contract Line Item Number* (CLIN) on that TO. Task orders contain one or more CLINs.
+    CLINs specify funding amounts and a time period in which funds can be spent.
+
+
+    Provisioned resources incur costs with participating CSPs. Costs related to provisioned resources can
+    be attributed to specific CLINs. Actual and forecasted CSP costs can be reported by CLIN.
   version: 0.1.0
-  title: ATAT External API - Provisioning
+  title: ATAT CSP API
   contact:
     email: atat-dev+provisioning_api@ccpo.mil
 tags:
-  - name: provisioning
+  - name: access
     description: >-
-      Operations to manage provisioned Portfolios and access to each.
-      Portfolios represent a collection of related Applications under the purview of a
-      Mission Owner.  Applications may consist of one or more Environments.
+      Operations to manage access to provisioned Portfolios
   - name: cost
     description: >-
       Operations to query for actual and forecasted costs
+  - name: provisioning
+    description: >-
+      Operations to manage provisioned Portfolios.
+      Portfolios represent a collection of related Applications under the purview of a
+      Mission Owner.  Applications may consist of one or more Environments.
 paths:
   '/portfolios':
     post:
```

Comment:
For the `description` of this operation:

```
      description: Adds a new Portfolio to be provisioned, including Applications, Environments, and Operators. Once complete, Operators should be able to log into the CSP and begin procuring all available services for their use, funded by the provided Funding Source(s). See '#/info/description' and '#/components/schemas' for details on these constructs.
```

Reply:
The comment here is on the description of the `provisioning` tag.  But I interpreted this to mean that the description of this tag could be simplified since we'd covered the detail here in the #info/description

Also, the longer description was applied to the description for operation `addPortfolio`.

Changed in b65917de3a6cb37d94e511d9fc97efa7ddc62b66.